### DISCUSSION TITLE
Playlist Song Arranging

### DISCUSSION THREAD

**Discussion by jdycus2558** [smplayer-dev/smplayer#760] [Q&A]
After shut down and rerun, songs in playlist appear in different order.

**Comment by Raza-0007** [comment]
It does not happen at my end. My playlist stays in the same order. 

You can sort the playlist by clicking on the columns inside the playlist, by track number, name and length, but  sometimes this conflicts with the following option "Options>Preferences>Playlist>Misc>Auto Sort". maybe that is what is messing up your sort order.

**Comment by Grooves1210** [comment]
Is `shuffle` turned on?
View --> Playlist  
The shuffle icon is a dice.  When it's on, it randomly moves the position of each file in the playlist
